src/main/java/seedu/address/logic/parser/FindPropertyCommandParser.java

```diff
@@ -39,13 +53,75 @@ public FindPropertyCommand parse(String args) throws ParseException {
         List<Predicate<Property>> predicates = new ArrayList<>();
         List<String> keywords = new ArrayList<>();
 
-        for (String s : nameKeywords) {
-            if (s.startsWith(String.valueOf(PREFIX_PROPERTY_PRICE_MORE))) {
-                predicates.add(new PropertyPricePredicate(s.split("/")[1], false));
-            } else if (s.startsWith(String.valueOf(PREFIX_PROPERTY_PRICE_LESS))) {
-                predicates.add(new PropertyPricePredicate(s.split("/")[1], true));
-            } else if (s.startsWith(String.valueOf(PREFIX_TYPE))) {
-                predicates.add(new PropertyTypePredicate(s.split("/")[1]));
+        for (int i = 0; i < nameKeywords.length; i++) {
+            String s = nameKeywords[i];
+            if (s.contains("/")) {
```

Comment:
actually, would using the ArgumentTokenizer be better?

Reply:
The default keywords don't have any prefixes though.

Reply:
Will adding `n/` for name be a solution? Since we are adding prefixes for the rest.

Reply:
Oh right eveything has prefix. Ok will go change the find name one to n/.